src/schemas/v2_route.json

```diff
@@ -2,7 +2,7 @@
   "name": "Route",
   "members": [
     {
-      "name": "name",
+      "name": "Intervention name",
```

Comment:
Right now these are used both as keys in the resulting JSON and in the form description. Maybe it's time to distinguish those two things already. For example, below, `FullSeparation` could become something like `{"value": "FullSeparation", "short_name": "Full separation", "description": "Solid raised kerb at least 1m in width separating the path from the carriageway. Eg Blackfriars road. Minimum width X meters. Should be used when possible. Costs about $123 per km"

Reply:
+1 to `value`, `short_name` and `description` keys in the JSON schema spec.